app/javascript/components/fonticon-picker/index.jsx

```diff
@@ -113,4 +113,4 @@ FontIconPicker.defaultProps = {
   },
 };
 
-export default FontIconPicker;
+export default FontIconPicker;
```

Comment:
Please ensure that all files have a trailing carriage return (the bot should flag this but it might not for .jsx files).  You may have to change your editor to make sure your editor does this.  This way, these changes won't be added to the PR.

Reply:
ok